packages/ui/src/components/Visualization/ContextToolbar/FlowExportImage/FlowExportImage.tsx

```diff
@@ -57,12 +51,19 @@ export function FlowExportImage() {
         onClick={onClick}
         variant="control"
         data-testid="exportImageButton"
+        isDisabled={isExporting}
       />
-
       {isExporting && (
-        <div className="export-overlay">
-          <div className="export-spinner"></div>
-        </div>
+        <>
+          <HiddenCanvas
+            entities={entitiesContext.visualEntities}
+            layout={currentLayout || LayoutType.DagreHorizontal}
```

Comment:
What do you think about allowing this to be `LayoutType | undefined`? So we can pass it directly from this component, and we assign the default value at the `HiddenCanvas` component:

```suggestion
            layout={currentLayout}
```